taskcluster/ci/config.yml

```diff
@@ -129,36 +105,12 @@ taskgraph:
             default-repository: https://github.com/mozilla-rally/rally-core-addon
             default-ref: release
             type: git
-        telemetrytestingextension:
-            name: "Telemetry testing extension"
-            project-regex: telemetrytestingextension$
-            default-repository: https://github.com/mozilla-extensions/dynamic-probe-telemetry-extension
-            default-ref: main
-            type: git
         searchdefaultaddon:
             name: "Reset Search Default extension"
             project-regex: searchdefaultaddon$
             default-repository: https://github.com/mozilla-extensions/restore-search-defaults
             default-ref: main
             type: git
-        quicksuggestweather:
-            name: "Firefox Quick Suggest Weather"
-            project-regex:  firefox-quick-suggest-weather$
-            default-repository: git@github.com:mozilla-extensions/firefox-quick-suggest-weather
-            default-ref: master
-            type: git
-        quicksuggestadm:
-            name: "Firefox Quick Suggest Adm"
-            project-regex:  firefox-quick-suggest-adm$
-            default-repository: git@github.com:mozilla-extensions/firefox-quick-suggest-adm
-            default-ref: master
-            type: git
-        quicksuggestcalculator:
-            name: "Firefox Quick Suggest Calculator"
-            project-regex:  firefox-quick-suggest-calculator$
-            default-repository: https://github.com/mozilla-extensions/firefox-quick-suggest-calculator.git
-            default-ref: master
-            type: git
```

Comment:
There are also some `normandy-privileged` lines below in this file that we can clean up, e.g. notifications and signing-format.